Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/PointerData/InteractablePointerData.cs

```diff
@@ -0,0 +1,149 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.UI
+{
+    /// <summary>
+    /// pointer information for tracking gestures and pokes
+    /// Abstacts different input types into a single basic set of data
+    /// These values can be accessed from themes or events to
+    /// to provide more granular feedback
+    /// </summary>
+    public struct InteractablePointerTransform
+    {
+        public Vector3 Position;
+        public Quaternion Rotation;
+        public Ray[] Rays;
+    }
+
+    public struct InteractablePressData
+    {
+        public Vector3 StartPosition;
+        public Vector3 Position;
+        public Vector3 Direction;
+        public float Distance;
+        public float Percentage;
+        public bool HasTouch;
+        public float MaxDistance;
+        public Vector3 ProjectedDirection;
+        public Vector3 PressedValue;
+    }
+
+    public class InteractablePointerData
```

Comment:
I don't understand what this class is used for, could you please explain via a comment?

Reply:
this class was removed. it was an attempt to capture generic gesture data.